subsys/debug/Kconfig

```diff
@@ -57,6 +57,20 @@ config ASAN
 	  This behavior can be changes by adding leak_check_at_exit=1 to the
 	  environment variable ASAN_OPTIONS.
 
+if ASAN
+config ASAN_NOP_DLCLOSE
+	bool "Use NOP dlclose"
+	default y if HAS_SDL
+	help
+	  Enable a NOP implementation of dlclose.
```

Comment:
Maybe better something like: "Override host OS dlclose() with a NOP" ?
(To be clearer to unsuspecting people)